src/session.js

```diff
@@ -26,6 +30,7 @@ Session.prototype.initialize = function(options) {
 
     visitor.notifyUnsyncedAssignments();
 
+    loaded(visitor);
```

Comment:
on line 18 above, we are setting `this._visitorLoaded` to the promise returned by `Visitor.loadVisitor`. I think we no longer need to do that with this change, since the promise defined in the constructor can continue to be used.